src/schema/v2/partner.ts

```diff
@@ -749,6 +749,11 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLBoolean,
         resolve: ({ verified_seller }) => verified_seller,
       },
+      wireTransferEnabled: {
```

Comment:
This might be not necessary for the Vibrations work because it directly consumes Gravity through Kinetic and hasn't integrated with Metaphysics yet.

For the checkout UI on the buyer side to get applicable payment methods, I wonder if we can expose something directly on an order itself.